.ci/azure-pipelines.yml

```diff
@@ -71,19 +73,20 @@ jobs:
           publishJUnitResults: false
           testResultsFiles: '**/TEST-*.xml'
           javaHomeOption: 'JDKVersion'
+          jdkVersionOption: 1.11
           sonarQubeRunAnalysis: false
         env:
           BINTRAY_KEY: $(bintray_key)
 
-      - task: GithubRelease@0
+      - task: GitHubRelease@1
```

Comment:
I can't find a changelog for this task but I'd like to know what the changes between 0 and 1 are to see if it doesn't contain breaking changes.

Reply:
TBH I was kind of surprised finding this too, but both v0 and v1 seem to be maintained.
There is sadly no changelog per say but here is the PR introducing [v1](https://github.com/microsoft/azure-pipelines-tasks/pull/11134)

Reply:
Should be no issue to update it when I read that PR. Weird how they maintain 2 separate tasks though.

Reply:
I know, but it seemed save enough + no idea how long v0 would continue maintenance